index.html

```diff
@@ -115,16 +129,19 @@
           <div class="hero-logo__img"></div>
           <h1 class="hero-logo__name">UNIHACK</h1>
         </div>
-        <p>Something something good!</p>
-        <p>- CEO of Bosch</p>
+        <div class="quote">
+          <p>Blown away by the great ideas at #UNIHACK2015!</p>
+          <p>Makes judging very difficult</p>
+        </div>
+        <p class="quote-from">- Hadi Micheal (SE at Deloitte)</p>
```

Comment:
Hadi Michel, not Hadi Michael. 

He's now a Product Manager at Facebook.

https://twitter.com/hdmchl